include/ocpp/v16/charge_point_configuration.hpp

```diff
@@ -86,6 +86,18 @@ class ChargePointConfiguration {
     KeyValue getSupportedChargingProfilePurposeTypesKeyValue();
     int32_t getMaxCompositeScheduleDuration();
     KeyValue getMaxCompositeScheduleDurationKeyValue();
+    std::optional<uint32_t> getCompositeScheduleDefaultLimitAmps();
+    std::optional<KeyValue> getCompositeScheduleDefaultLimitAmpsKeyValue();
+    void setCompositeScheduleDefaultLimitAmps(uint32_t limit_amps);
+    std::optional<uint32_t> getCompositeScheduleDefaultLimitWatts();
+    std::optional<KeyValue> getCompositeScheduleDefaultLimitWattsKeyValue();
+    void setCompositeScheduleDefaultLimitWatts(uint32_t limit_watts);
+    std::optional<uint32_t> getCompositeScheduleDefaultNumberPhases();
+    std::optional<KeyValue> getCompositeScheduleDefaultNumberPhasesKeyValue();
+    void setCompositeScheduleDefaultNumberPhases(uint32_t number_phases);
```

Comment:
Since the number of phases arguably never exceed 3 this won't really be an issue, but in the 1.6 spec number of phases (like all other integers) are signed 32 bit integers